2.6/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -151,8 +147,10 @@ function mongo_primary_member_addr() {
 
 # mongo_remove removes the current MongoDB from the cluster
 function mongo_remove() {
-  local primary_addr
-  primary_addr="$(mongo_primary_member_addr)"
+  local primary_addr="$(mongo_primary_member_addr ||  true)"
```

Comment:
@php-coder how about we get this back to 2 lines with the explicit `|| true`?
It would be nice to have a comment/reminder as to why we are ignoring the exit code.

In this particular line here you have an extra space between the `||` and `true`.